src/ui/widgets/config/WidgetConfigForm.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { intlShape } from 'react-intl';
 import Form from 'react-jsonschema-form';
+import apps from 'entando-apps';
 
-const WidgetConfigForm = ({ onSubmit, widgetConfig }) =>
-  (widgetConfig && widgetConfig.schema ? (
+const widgetForms = apps.reduce((obj, app) => ({
```

Comment:
This shouldn't be coupled with CMS. It should be an integration point for every kind of appbuilder app